dev/sbt-plugin/src/sbt-test/sbt-plugin/run-ssl/test

```diff
@@ -0,0 +1,17 @@
+# Structure of this test:
```

Comment:
Should we also add a check if the trust store file exists?

Reply:
I didn't want to assert an implementation detail. Could be convinced otherwise. :-)

Reply:
Nah, I have the same doubt. :-)